### ISSUE TITLE
E8a/E9a - Markup Validation failed

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#150]
Events by Month & Week both have errors in the Markup Validator

[Events by Month](https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.web.uoguelph.ca%2Foac%2Fevents%2Fmonth)
[Events by Week](https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.web.uoguelph.ca%2Foac%2Fevents%2Fweek)

This may be related to Issue #142

**Comment by mmafe** [comment]
Fixed as part of hotfix20150807.3.0

**Comment by jpinsent** [comment]
Looks like it's working. No issues on AODA Demo or OAC validation checks.
